identity_ntp/recipes/default.rb

```diff
@@ -0,0 +1,20 @@
+package 'ntp' do
+  action :upgrade
+end
+
+template '/etc/ntp.conf' do
+  source 'ntp.conf.erb'
+  owner 'root'
+  group 'root'
+  mode 0644
+end
+
+service 'ntp' do
+  action [ :enable, :start ]
+end
+
+execute 'restart-ntp-if-no-peers' do
+  command 'true'
+  not_if "/usr/bin/ntpdc -c peers | egrep -v '============|remote.*local.*st' | awk '{print $5}' | grep -v 0"
```

Comment:
I don't think this command works. We should probably remove it. (Not sure it has ever worked, honestly.)

```
# /usr/bin/ntpdc -n -c peers
localhost: timed out, nothing received
***Request timed out
```